src/read_write/codec.rs

```diff
@@ -87,7 +87,7 @@ where
     num::cast(value).unwrap()
 }
 
-pub fn encode<T>(value: f64, min: f64, edge_length: f64) -> T
+pub fn _encode<T>(value: f64, min: f64, edge_length: f64) -> T
```

Comment:
You could also do `#[allow(dead_code)]` here. Not sure which one is nicer.

Reply:
I feel like _encode calls attention to this function being unused, which is nice